README.md

```diff
@@ -116,26 +102,23 @@ other distributions are requested, pbuidler chroots are created on the fly.
 These chroots are lost when the container is removed, but only take 7.5sec to
 create.
 
-## Docker Image Architecture
+## Docker image architecture
```

Comment:
This whole section is out of date now. I forgot to update it when I re-architected the image. You can remove this section.